### PR TITLE
Add cisagov/ansible-role-geoip2 to the Packer configuration

### PR DESCRIPTION
**Pull request by mcdonnnj**
#  #

## 🗣 Description ##

This pull request adds [cisagov/ansible-role-geoip2] as an Ansible requirement for our Packer configurations and alters the usage of [cisagov/ansible-role-cyhy-core] to pass the MaxMind GeoIP2 license as a role variable.

**Note**: This requires https://github.com/cisagov/ansible-role-cyhy-core/pull/54 to be merged first.

## 💭 Motivation and context ##

This puts [cisagov/ansible-role-cyhy-core] in line with our development practices to take a role variable instead of looking up a value itself. This is also part of the move to separate the MaxMind GeoIP2 database installation into its own role for separate use.

## 🧪 Testing ##

Automated testing passes. I have this used to build and AMIs in my testing workspace and I have verified that it works as expected.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All new and existing tests pass.

[cisagov/ansible-role-cyhy-core]: https://github.com/cisagov/ansible-role-cyhy-core
[cisagov/ansible-role-geoip2]: https://github.com/cisagov/ansible-role-geoip2

### CODE REVIEW

File: packer/ansible/playbook.yml [hunk 1]
```diff
@@ -95,10 +99,14 @@
...
+    maxmind_license_key: "{{ lookup('aws_ssm', '/cyhy/core/geoip/license_key', region='us-east-1') }}"
 
 - hosts: cyhy_archive
   name: Install cyhy-archive helper script
   become: yes
   become_method: sudo
   roles:
     - cyhy_archive
+  vars:
+    maxmind_license_key: "{{ lookup('aws_ssm', '/cyhy/core/geoip/license_key', region='us-east-1') }}"
```
Comment (dav3r) [inline packer/ansible/playbook.yml:112]: Why do we need the maxmind key here?

**Review by dav3r — COMMENTED** [review]
This looks good, but I have one question before I approve...

File: packer/ansible/playbook.yml (hunk 1 above)
Comment (mcdonnnj) [reply packer/ansible/playbook.yml:112]: Dependency chain. [cisagov/ansible-role-cyhy-archive](https://github.com/cisagov/ansible-role-cyhy-archive) depends on [cisagov/ansible-role-cyhy-core](https://github.com/cisagov/ansible-role-cyhy-core) which depends on [cisagov/ansible-role-geoip2](https://github.com/cisagov/ansible-role-geoip2).

Comment (dav3r) [reply packer/ansible/playbook.yml:112]: Thanks for the reminder! 👍

**Review by dav3r — APPROVED** [review]
👍 👍

**Review by felddy — APPROVED** [review]
🇺🇦 ☮️ 👍

**Review by jsf9k — APPROVED** [review]
_I love it!!!_

**Pull request merged**